Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -79,7 +79,7 @@ class MainTabBarViewController: UITabBarController {
             selector: #selector(handleSafeCreated),
             name: .safeCreationUpdate,
             object: nil)
-
+        
```

Comment:
this adds not needed spaces. Please check that your Xcode has "trim spaces" enabled
![Screenshot 2022-02-28 at 15 40 52](https://user-images.githubusercontent.com/284056/156002147-ebc9cf9b-8ab8-45f9-b0e7-a8cb2f0241cf.png)
: